integration_tests/test_cli.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import pandas as pd
+import os
+
+os.makedirs('integration_tests/outputs/raw', exist_ok=True)
+os.makedirs('integration_tests/outputs/hamronized', exist_ok=True)
+
+def perform_cli_test(tool, file, folder, db=None):
+    command = ['argnorm', tool, '-i', f'examples/{folder}/{file}', '-o', f'integration_tests/outputs/{folder}/{file}']
+    
+    if folder == 'hamronized':
+        command.append('--hamronized')
+
+    if tool == 'abricate':
+        command += ['--db', db]
+
+    subprocess.check_call(command)
+
+    output = pd.read_csv(f'integration_tests/outputs/{folder}/{file}', sep='\t')
+    golden_file = pd.read_csv(f'outputs/{folder}/{file}', sep='\t')
+    pd.testing.assert_frame_equal(output, golden_file)
+
+def test_argsoap_cli(folder):
+    file = 'args-oap.sarg.reads.tsv'
+    perform_cli_test('argsoap', file, folder)
+
+test_argsoap_cli('hamronized')
+test_argsoap_cli('raw')
+
+def test_deeparg_cli(folder):
```

Comment:
1. This looks too much like a pytest-style test (when it isn't)
2. What is even the point of having a separate function instead of just having the code:

```python
for folder in ['hamronized', 'raw']:
    perform_cli_test('argsoap', 'args-oap.sarg.reads.tsv', folder)
```

Reply:
I have just updated the code without the separate functions. Does this look ok?